datastore/save.go

```diff
@@ -53,6 +54,32 @@ func reflectFieldSave(props *[]Property, p Property, name string, opts saveOpts,
 	switch x := v.Interface().(type) {
 	case *Key, time.Time, GeoPoint:
 		p.Value = x
+	case civil.Date, civil.DateTime, civil.Time:
```

Comment:
I would just separate each of these cases given that you are effectively handling them completely separately below.

Reply:
Whoops -- duplicated comment. +1.